tools/test_output.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
```

Comment:
I seems like all the tests in this file test nearly the same thing. Could this be condensed?